README.md

```diff
@@ -1,2 +1,28 @@
-# bankrupt-officer-search-api
-API for searching for bankrupt officers
+<!-- FIXME update API name -->
+# Companies House Bankrupt Officer Search API
+<!-- FIXME update API name -->
+## bankrupt-officer-search-api
+API handling bankrupt officer search
+
+### Requirements
+* [Java 8][1]
+* [Maven][2]
+* [Git][3]
+
+### Getting Started
+1. Run `make` to build
+2. Run `./start.sh` to run
+
+### Environment Variables
+Name | Description | Mandatory | Location
+--- | --- | --- | ---
+BANKRUPT_OFFICER_SEARCH_API_PORT | Port this application runs on when deployed. | ✓ | start.sh
```

Comment:
This is only relevant for Vagrant, you can remove this from the README

Reply:
Should I remove this or specify that it is Vagrant Environment Variables?